Makefile

```diff
@@ -30,11 +30,11 @@ SPEC_SOURCES := $(shell find spec -name '*.cr')
 override FLAGS += $(if $(release),--release )$(if $(stats),--stats )$(if $(progress),--progress )$(if $(threads),--threads $(threads) )$(if $(debug),-d )$(if $(static),--static )$(if $(LDFLAGS),--link-flags="$(LDFLAGS)" )
 SPEC_WARNINGS_OFF := --exclude-warnings spec/std --exclude-warnings spec/compiler
 SPEC_FLAGS := $(if $(verbose),-v )$(if $(junit_output),--junit_output $(junit_output) )
+CRYSTAL_CONFIG_LIBRARY_PATH := $(shell crystal env CRYSTAL_LIBRARY_PATH)
```

Comment:
Should this be `bin/crystal`? Both to make sure we get any changed output while developing without having to recompile twice, but also so you can bootstrap by just placing a cross compiled binary at `.build/crystal` with no further setup.

Reply:
In any case this should have a `|| ""` to suppress an error message when building without a `crystal` in `$PATH`.

Reply:
Sounds right. I don't think you are missing something. Going through bin/crystal will find crystal in the path.

Reply:
Also, something that is not quite tidy yet is that libcrystal is not located in the embedded folder.  Eventually this setting should put before the path where libcrystal is generated (instead of the current path within the std-lib). But that will be in the future.

Reply:
I'm not sure. It's just bothers me that libcrystal is between the .cr files. Not big deal for now, but is related.